internal/command/deploy/machines.go

```diff
@@ -544,7 +544,7 @@ func (md *machineDeployment) setStrategy(passedInStrategy string) error {
 	return nil
 }
 
-func (md *machineDeployment) createReleaseInBackend(ctx context.Context) error {
+func CreateReleaseInBackend(ctx context.Context, client *api.Client, app *appv2.Config, strategy, image string) (*gql.MachinesCreateReleaseResponse, error) {
```

Comment:
how about moving this to the api package, since we're making it more general use?

Reply:
I usually try to avoid naming things `app`, since we have an `app` package that gets used in a bunch of places. Not a big deal here, but that's why we have appConfig in a bunch of place.

Reply:
moving it to the `api` package solves the second one cleanly too! :)